antarest/core/utils/utils.py

```diff
@@ -24,17 +24,31 @@
 from pathlib import Path
 
 import py7zr
-import redis
 from fastapi import HTTPException
+from pydantic import BaseModel, ConfigDict
 
-from antarest.core.config import RedisConfig
 from antarest.core.exceptions import ShouldNotHappenException
 
 logger = logging.getLogger(__name__)
 
 UUID_PATTERN = re.compile("^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$")
 
 
+class BaseModelInHouse(BaseModel):
```

Comment:
Proposition: renaming to AntaresBaseModel and move it to `antares/core/serialization`